bin/rolling_index

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+
+require_relative '../config/environment'
+require 'daemons'
+
+QUERY = { q: '*:*', sort: 'timestamp asc', fl: 'id', rows: '500' }
+
+Daemons.run_proc('rolling index') do
+  loop do
+    solr_conn = RSolr.connect(timeout: 120, open_timeout: 120, url: Settings.solrizer_url)
+    indexer = Indexer.new(solr: solr_conn)
+
+    response = solr_conn.get 'select', params: QUERY
+    response['response']['docs'].each do |doc|
+      puts "Doc #{doc}"
+      indexer.reindex_pid(doc['id'], add_attributes: { commitWithin: 1000 })
+    end
+    sleep(5)
+  end
+end
```

Comment:
Hi! This looks solid to me. I'm curious about reaching for `daemons` rather than, say, a scheduled task. Can you say more about that?

Reply:
This is constantly running, so it's a daemon. It's not scheduled.

Reply:
@jcoyne A benefit of the daemon over the scheduled task is lower/no risk of one task running and not completing before the next task is invoked. A cost is needing to make sure this process is run at start time and boot time and restarted at deploy time. Do we need any sort of process management on top to help manage this?

Reply:
Yeah, I think we'll need to work with ops to get a systemd(?) script to run this.

Reply:
@jcoyne and we'll prolly also need some capistrano work to use said systemd script. Would you mind ticketing if not already done? That assuages my concern. Approving.

Reply:
Sure I will do that.